### ISSUE TITLE
KNC: JKZD/JKNZD valid with VEX.L=1

### ISSUE THREAD

**Issue by flobernd** [intelxed/xed#282]
Hi there,

an user reported the following problem to us:

In the Intel® Xeon Phi™ ISA manual, the `JKZD/JKNZD` instructions are defined as:

```
VEX.NDS.128.0F.W0 85 id jknzd k1, rel32
VEX.NDS.128.W0 75 ib    jknzd k1, rel8
```

Currently xed (and Zydis as well) successfully decodes these instructions even with `VEX.L=1` regardless of the `VEX.L=0 / 128` constraint in the above definition.

As we don't have actual hardware to test, we are now struggling to decide wether the KNC ISA manual or the XED datafiles are correct here. It would be great if you could help us out here :-)

_Original issue:_
https://github.com/zyantific/zydis/issues/294

**Comment by marjevan** [comment]
VEX.L should be 0.
We'll fix XED accordingly.

Thank you for your report.

**Comment by flobernd** [comment]
Thanks for the clarification. We will update `Zydis` as well.

**Comment by marjevan** [comment]
Fixed (External release v2022.04.17).
